schema/gks-core/gks-core-source.yaml

```
@@ -188,26 +188,40 @@ $defs:
     description: >-
       A structured representation of a code for a defined concept in a terminology or code system.
     properties:
+      code:
+        $ref: '#/$defs/code'
+        description: >-
+          A symbol uniquely identifying the concept, as in a syntax defined by the code system.
+          CURIE format is preferred where possible (e.g. 'SO:0000704' is the CURIE form of the
+          Sequence Ontology code for 'gene'). See https://www.w3.org/TR/curie/.
```

@mbrush "A symbol uniquely identifying the concept, as in a syntax defined by the code system." and "CURIE format is preferred where possible" seem to conflict. For example, NCIt code syntax is `C\d+`. But to me, the CURIE format part could mean that we should use `ncit:C\d+`. 

A [previous description version](https://github.com/ga4gh/gks-core/blame/5fbd21ba7f40531018fe012b22e8e0fa445760f5/schema/gks-common/json/Coding#L23) states that code should be `"Symbol in syntax defined by the terminology system."`

I think we should consider updating the description to:

```suggestion
          A symbol uniquely identifying the concept, as in a syntax defined by the code system.
          (e.g. 'SO:0000704' is the proper code syntax defined by Sequence Ontology for 'gene'
          and 'C287' is the proper code syntax defined by NCIt for 'aspirin').
```

Tagging @ahwagner @larrybabb for comment

---

Thanks Kori.  I think a broader discussion is needed around if and how strongly we recommend the use of CURIEs - so we can address issues like the one you raise, and others that have come up due to our trying to support use of CURIES in a FHIR-like Coding object that was not designed to support them. This will definitely be on the agenda to address over the next few weeks as we finalize things for the 1.0 release.